db/redis.js

```diff
@@ -64,7 +64,7 @@ async function addDoorTestSensorData(locationid, signal) {
 async function addIM21DoorSensorData(locationid, doorSignal) {
   // eslint-disable-next-line eqeqeq
   if (doorSignal == SESSIONSTATE_DOOR.CLOSED || doorSignal == SESSIONSTATE_DOOR.OPEN) {
-    await client.xadd(`door:${locationid}`, '*', 'signal', doorSignal)
+    await client.xadd(`door:${locationid}`, 'MAXLEN', '~', '10000', '*', 'signal', doorSignal)
```

Comment:
Are these maximums for the single location or for the whole redis instance?

Reply:
for a single location. The way it works is that it limits the length of a key, in this case a single key would be door:TestLocation

Reply:
Gotcha. Thanks for clarifying.